mysite/settings.py

```diff
@@ -137,6 +137,9 @@
 
 SITE_ID = 1
 
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```

Comment:
Later on we want to have console only `if DEBUG` and a real provider if not debug (e.g. you can provision sendgrid on Heroku)